modules/integration/tests-common/integration-tests-utils/src/main/java/org/wso2/mb/integration/common/utils/ui/pages/main/TopicSubscriptionsPage.java

```diff
@@ -84,6 +85,84 @@ public int getDurableInActiveSubscriptionsCount() {
     }
 
     /**
+     * Search topic subscriptions according to the search criteria.
+     *
+     * @param queueNamePattern string pattern of the topic name (* for all)
+     * @param identifierPattern string pattern of the identifier (* for all)
+     * @param ownNodeIdIndex index of the node Id in the dropdown the subscriptions belong to
+     * @return number of subscriptions listed under search result
+     */
+    public void searchTopicSubscriptions(String queueNamePattern, String identifierPattern, int
+            ownNodeIdIndex, boolean isNameExactMatch, boolean isIdentifierExactMatch) {
+
+        WebElement queueNamePatternField = driver.findElement(By.name(UIElementMapper.getInstance()
+                .getElement("mb.search.topic.name.pattern.tag.name")));
+        queueNamePatternField.clear();
+        queueNamePatternField.sendKeys(queueNamePattern);
+
+        WebElement queueIdentifierPatternField = driver.findElement(By.name(UIElementMapper.getInstance()
+                .getElement("mb.search.topic.identifier.pattern.tag.name")));
+        queueIdentifierPatternField.clear();
+        queueIdentifierPatternField.sendKeys(identifierPattern);
+
+        //Check/ uncheck the Topic Name exact match check box
+        WebElement topicNameExactMatchField = driver.findElement(
+                By.name(UIElementMapper.getInstance().getElement("mb.search.topic.name.exactmatch.tag.name")));
+
+        if (isNameExactMatch && !topicNameExactMatchField.isSelected()) {
+            topicNameExactMatchField.click();
+        }
+        if (!isNameExactMatch && topicNameExactMatchField.isSelected()) {
+            topicNameExactMatchField.click();
+        }
+
+        //Check/ uncheck the Topic Identifier exact match check box
+        WebElement topicIdentifierExactMatchField = driver.findElement(
+                By.name(UIElementMapper.getInstance().getElement("mb.search.topic.identifier.exactmatch.tag.name")));
+
+        if (isIdentifierExactMatch && !topicIdentifierExactMatchField.isSelected()) {
+            topicIdentifierExactMatchField.click();
+        }
+        if (!isIdentifierExactMatch && topicIdentifierExactMatchField.isSelected()) {
+            topicIdentifierExactMatchField.click();
+        }
+
```

Comment:
I was talking about both of the if statements. The review view only shows one if statement :)